docs/config_search.md

```diff
@@ -61,12 +63,15 @@ Model Analyzer's default search mode depends on the type of model and if you are
 - [Ensemble Model Search](config_search.md#ensemble-model-search):
   - **Default Search type:** [Quick Search](config_search.md#quick-search-mode)
   - **Command:** N/A
+- [BLS Model Search](config_search.md#bls-model-search):
+  - **Default Search type:** [Quick Search](config_search.md#quick-search-mode)
+  - **Command:** N/A
```

Comment:
I don't like that command is listed for all 4 but only one of them actually has that information.

Reply:
Removed